app/serializers/reviewable_score_serializer.rb

```diff
@@ -78,10 +78,15 @@ def watched_word_reason(link)
     if object.context.nil?
       # If the words weren't recorded, try to guess them based on current settings.
       words =
-        WordWatcher.new(object.reviewable.post.raw).word_matches_for_action?(
-          :flag,
-          all_matches: true,
-        )
+        if object.reviewable.respond_to?(:post)
+          WordWatcher.new(
```

Comment:
Super minor, but maybe it'd be cleaner to have `WordWatcher#initialize` take variadic arguments and smush them internally so each call site doesn't have to do this string interpolation.

Reply:
Aye, that could help in some instances, but I suspect it would end up a different kind of messy when the strings are conditionally included.